lib/config.py

```diff
@@ -200,6 +200,10 @@ def _setup_config_parser_options(self):
         self.parser.add_argument('--log-size',
                                  help='Size in bytes above which the log file '
                                  'should rotate', type=int, default=2<<20)
+        self.parser.add_argument('--work-dir', '-w',
+                                 help='Directory used to store all artifacts '
```

Comment:
- I'd avoid using Jenkins terminology (artifacts) in host_os.py
- Make it explicit this directory contains *temporary* files

I suggest "artifacts" => "temporary files"